Revise the API reference pages in the docs (#142)
ihmeuw/pseudopeople: docs/source/api_reference/index.rst

API Reference
=============

This section of the docs describes the details of how to use each function included in pseudopeople.
These functions make up pseudopeople's public Application Programming Interface (API).

> If you are a new user, don't start here!
> Start with the :ref:`Quickstart <quickstart>`. 

Reviewer: ```suggestion
If you are a new user, you may want to start with the :ref:`Quickstart <quickstart>` guide instead. 
```
